### PR TITLE
docs: add docs on data flows and datamodels for staging areas

### PR DESCRIPTION
**Pull request by BrendaHijmans**
* documentation on data flows in folder docs
* data models for staging areas and explanation how to set up staging areas in folder datamodels

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ the catalogue.
 4. It extracts all data from the catalogue and loads the networks' (or models') variable metadata to the staging 
 area catalogue (containing only the common data models).
 
+See folder docs for more details on data flows.
```
Comment (connoratrug) [inline README.md:10]: i think you can add a link in the mark down to the folder

File: docs/data-flows-catalogue.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#Data flows for data catalogue staging areas
+
+We use staging areas for data managers to upload their network or cohort variable metadata and cohort mappings.
+Data managers of cohorts get access to databases on a server to upload their data using templates in a preloaded model, which is a derivative of the [full data catalogue model](https://github.com/molgenis/molgenis-emx2/blob/master/data/datacatalogue/molgenis.csv): 
+Links to the models and how to set up staging areas for the data cataloge are [here](https://github.com/molgenis/molgenis-py-catalogue-transform/datamodels)
```
Comment (connoratrug) [inline docs/data-flows-catalogue.md:5]: suggest to move the link from 'here' to 'models and how to set up staging areas for the data cataloge' to allow use the scan for links

File: docs/data-flows-catalogue.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+Catalogue – The data in tables Resources, Networks, Tables, Variables, VariableValues, RepeatedVariables from all Networks and Models in PRODUCTION catalogue. Data flow is from PRODUCTION to STAGING. All releases are taken out of the model.
+
+CatalogueOntologies – All OntologyTerms tables. Data flow is from STAGING to PRODUCTION, so that changes can be made in STAGING and then copied to PRODUCTION. Changes are made on request by MOLGENIS data managers.
+
+Networks – e.g. LifeCycle, ATHLETE, LongITools. Central data managers fill out CDM here. Versioning is handled by central data managers. Data flow is from STAGING to PRODUCTION. The data model can be found here: https://drive.google.com/drive/folders/1zts1USXV78CB-4bnxQlIFEGCKoEZ_oB5, network_model.zip
+
+Cohorts – e.g. BIB, ALSPAC. Cohort data managers fill out dictionary and mappings to a or more CDMs here. Versioning is not applied (for now all versions are set at 1.0.0). Data flow is from STAGING to PRODUCTION. The data model can be found here: https://drive.google.com/drive/folders/1zts1USXV78CB-4bnxQlIFEGCKoEZ_oB5, cohort_model.zip
+
+
+PRODUCTION contains one database:
```
Comment (connoratrug) [inline docs/data-flows-catalogue.md:19]: suggest avoiding the term 'Production' as the copy of this flow could be setup in test or dev mode. suggest using the term 'catalogue'

File: docs/data-flows-catalogue.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+#Databases
+
+STAGING can contain four different types of ‘databases’:
+Catalogue – The data in tables Resources, Networks, Tables, Variables, VariableValues, RepeatedVariables from all Networks and Models in PRODUCTION catalogue. Data flow is from PRODUCTION to STAGING. All releases are taken out of the model.
+
+CatalogueOntologies – All OntologyTerms tables. Data flow is from STAGING to PRODUCTION, so that changes can be made in STAGING and then copied to PRODUCTION. Changes are made on request by MOLGENIS data managers.
+
+Networks – e.g. LifeCycle, ATHLETE, LongITools. Central data managers fill out CDM here. Versioning is handled by central data managers. Data flow is from STAGING to PRODUCTION. The data model can be found here: https://drive.google.com/drive/folders/1zts1USXV78CB-4bnxQlIFEGCKoEZ_oB5, network_model.zip
+
+Cohorts – e.g. BIB, ALSPAC. Cohort data managers fill out dictionary and mappings to a or more CDMs here. Versioning is not applied (for now all versions are set at 1.0.0). Data flow is from STAGING to PRODUCTION. The data model can be found here: https://drive.google.com/drive/folders/1zts1USXV78CB-4bnxQlIFEGCKoEZ_oB5, cohort_model.zip
```
Comment (connoratrug) [inline docs/data-flows-catalogue.md:16]: hard to read , suggest using a (numbered)list 

... 4 types: 

1. Foo 
2. Bar

**Comment by sidohaakma** [comment]
[Documentation data catalogue data flows on github + datamodels](https://trello.com/c/XK4HnLVb/237-documentation-data-catalogue-data-flows-on-github-datamodels)

**Pull request merged**